contents/docs/surveys/destinations.mdx

```diff
@@ -29,4 +29,5 @@ You can extract the following properties from surveys:
 | `event.properties.$survey_name` | The name of the survey |
 | `event.properties.$survey_questions` | An array of objects containing the following survey question properties: `id`, `index`, and `question` |
 | `event.properties.$survey_response_{response_key}` | The response to a specific question. To find `response_key`, go to your survey page and click on the button **Copy survey response key**. There's one for each question in your survey. `response_key` is identical to the corresponding survey question ID |
-| `person.name` or other person properties | The name of the person who responded or other properties |
\ No newline at end of file
+| `event.properties.$survey_completed` | Whether the survey was completed. Relevant for surveys with partial responses. This will be true if the user has answered all questions. |
```

Comment:
```suggestion
| `event.properties.$survey_completed` | Whether the survey was completed. Relevant for surveys with partial responses. This will be `true` if the user has answered all questions. |
```